.githooks/pre-push

```diff
@@ -7,13 +7,13 @@ RED='\033[0;31m'
 NC='\033[0m' # No Color
 
 # Check that prettier formatting rules are not violated.
-if which zk_supervisor >/dev/null; then
-  if ! zk_supervisor fmt --check; then
+if which zkstack >/dev/null; then
+  if ! zkstack dev fmt --check; then
```

Comment:
I think you still need to keep old code in hooks, because people have `zk_supervisor` installed. If it's installed, allow proceeding, but put a **big warning in red** to remove zki/zks/zkt/zkup and install zkstackup and zkstack instead.

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3003/commits/d84ad52e610aadbd0a393581d12b2785d37c5fd7

Q: is `zkt` deprecated too? Currently, the task description does not describe any change concerning `zkt`. @Deniallugo